stix_shifter_modules/error_test/entry_point.py

```diff
@@ -0,0 +1,12 @@
+from stix_shifter_utils.utils.base_entry_point import BaseEntryPoint
+
+
+class EntryPoint(BaseEntryPoint):
+
+    def __init__(self, connection={}, configuration={}, options={}):
+        super().__init__(connection, configuration, options)
+        self.set_async(False)
+        if connection:
+            self.setup_transmission_basic(connection, configuration)
+
+        self.add_dialect('default', default=True)
```

Comment:
you can just set this instead `self.setup_translation_simple(dialect_default='default')` to be consistent with other connectors.

Reply:
Am I replacing the entire `self.setup_transmission_basic(connection, configuration)` here? Won't I still need to pass in connection and configuration?

Reply:
sorry should have mentioned, this is just to replace `self.add_dialect('default', default=True)`

Reply:
or you can just leave it. which should also works.